projects/uitdatabank/docs/search-api/advanced/advanced-queries.md

```diff
@@ -958,6 +958,45 @@ Retrieve all events and places that have 5 to 10 videos:
 GET /offers/?q=videosCount:[5 TO 10]
 ```
 
+### completeness
+
```

Comment:
```suggestion
<!-- theme: info -->

> #### New functionality 🚧
>
> `completeness` is a new parameter that is still under construction. At the moment the completeness is only calculated for events, places and organizers created after February 15, 2024. 
```

Reply:
I would add this disclaimer so we can release the documentation for the parameter. We can then remove it once the dataset of UiTdatabank has been replayed